src/llaminator.scss

```diff
@@ -49,3 +49,7 @@ llama-select-fab {
   bottom: 32px;
   right: 32px;
 }
+
+.mdc-card__media {
```

Comment:
Instead of changing `.mdc-card__media`, which is not our class, we should instead add another class of our own. As long as it's listed _after_ `.mdc-card__media` in the `@class` attribute, that'll work as expected.

Reply:
done